### ISSUE TITLE
Ensure links are distinguished from surrounding text in a way that does not rely on color (a[href$="license"])

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/vscode-docs#7368]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**
### GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #Chrome; #WCAG1.4.1; #WCP; #Use of Color; #Visual Studio Code Services; #A11yAuto; #AI4W; #BM_VisualStudioCodeServices_Web_Apr2023; #Win11;#clink-in-text-block;
### Environment Details: 
Application Name:  code.visualstudio.com
URL: https://code.visualstudio.com/download?dark-plus-v2
Microsoft Edge version 125.0.0.0 (Official Build) (64-bit)
### Repro Steps:

1. Hit the URL: https://code.visualstudio.com/download?dark-plus-v2
2. Run fast pass and observe the issue.

### Issue: 
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.8/link-in-text-block?application=msftAI)
### Target application: 
Download Visual Studio Code - Mac, Linux, Windows - https://code.visualstudio.com/download?dark-plus-v2
### Element path: 
a[href$="license"]
### Snippet:
<a href="/license">license terms</a>
### How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 1.02:1 with the surrounding text. (Minimum contrast is 3:1, link text: #4daafc, surrounding text: #9ba3b4)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 125.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.42.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Attachment:
![image](https://github.com/microsoft/vscode-docs/assets/172380684/146ee223-eeed-4308-a6ed-4d30cf0c1e91)

**Comment by rzhao271** [comment]
There are underlines now, and the issue should be fixed.

**Comment by meganrogge** [comment]
@Govardhani-T could you please retest and close this issue if all works well?

**Comment by Govardhani-T** [comment]
Verified the issue in the environment. Issue is fixed.
![image](https://github.com/user-attachments/assets/ea212c55-b6e4-4fc3-a4b5-6aec320fea30)
